radio_beam/beam.py

```diff
@@ -386,7 +386,11 @@ def __eq__(self, other):
         this_pa = self.pa.to(u.deg) % (180.0 * u.deg)
         other_pa = other.pa.to(u.deg) % (180.0 * u.deg)
 
-        equal_pa = True if np.abs(this_pa - other_pa) < atol_deg else False
+        if self.iscircular():
+            equal_pa = True
+        else:
+            equal_pa = True if np.abs(this_pa - other_pa) < atol_deg else False
```

Comment:
Edge case here, but two beams can be the same and have PA offset by 180°.

Reply:
`pa % 180` is taken above. But we weren't explicitly testing this case, so I added it.